firefox_ios/views/unified_metrics.view.lkml

```diff
@@ -0,0 +1,738 @@
+    sql: ${TABLE}.metadata.isp.organization ;;
+    group_label: "Metadata ISP"
+    group_item_label: "Organization"
+  }
+
+  dimension: metadata__user_agent__browser {
+    type: string
+    sql: ${TABLE}.metadata.user_agent.browser ;;
+    group_label: "Metadata User Agent"
+    group_item_label: "Browser"
+  }
+
+  dimension: metadata__user_agent__os {
+    type: string
+    sql: ${TABLE}.metadata.user_agent.os ;;
+    group_label: "Metadata User Agent"
+    group_item_label: "OS"
+  }
+
+  dimension: metadata__user_agent__version {
+    type: string
+    sql: ${TABLE}.metadata.user_agent.version ;;
+    group_label: "Metadata User Agent"
+    group_item_label: "Version"
+  }
+
+
+  dimension: normalized_app_name {
+    type: string
+    sql: ${TABLE}.normalized_app_name ;;
+  }
+
+  dimension: normalized_channel {
+    type: string
+    sql: ${TABLE}.normalized_channel ;;
+  }
+
+  dimension: normalized_country_code {
+    type: string
+    sql: ${TABLE}.normalized_country_code ;;
+  }
+
+  dimension: normalized_os {
+    type: string
+    sql: ${TABLE}.normalized_os ;;
+  }
+
+  dimension: normalized_os_version {
+    type: string
+    sql: ${TABLE}.normalized_os_version ;;
+  }
+
+  dimension: ping_info__end_time {
+    type: string
+    sql: ${TABLE}.ping_info.end_time ;;
+    group_label: "Ping Info"
+    group_item_label: "End Time"
+  }
+
+  dimension: ping_info__experiments {
+    hidden: yes
+    sql: ${TABLE}.ping_info.experiments ;;
+    group_label: "Ping Info"
+    group_item_label: "Experiments"
+  }
+
+  dimension: ping_info__ping_type {
+    type: string
+    sql: ${TABLE}.ping_info.ping_type ;;
+    group_label: "Ping Info"
+    group_item_label: "Ping Type"
+  }
+
+  dimension: ping_info__reason {
+    type: string
+    sql: ${TABLE}.ping_info.reason ;;
+    group_label: "Ping Info"
+    group_item_label: "Reason"
+  }
+
+  dimension: ping_info__seq {
+    type: number
+    sql: ${TABLE}.ping_info.seq ;;
+    group_label: "Ping Info"
+    group_item_label: "Seq"
+  }
+
+  dimension: ping_info__start_time {
+    type: string
+    sql: ${TABLE}.ping_info.start_time ;;
+    group_label: "Ping Info"
+    group_item_label: "Start Time"
+  }
+
+  dimension: sample_id {
+    type: number
+    sql: ${TABLE}.sample_id ;;
+  }
+
+  dimension_group: submission_timestamp {
+    type: time
+    timeframes: [
+      raw,
+      time,
+      date,
+      week,
+      month,
+      quarter,
+      year
+    ]
+    sql: ${TABLE}.submission_timestamp ;;
+  }
+
+  # This is required, otherwise there will be duplicate data
+  dimension: telemetry_system {
+    type: string
+    sql: ${TABLE}.telemetry_system ;;
+  }
+
+  dimension: metrics__counter__glean_validation_foreground_count {
+    hidden: yes
+    type: number
+    sql: ${TABLE}.metrics.counter.glean_validation_foreground_count ;;
+    group_label: "Metrics Counter"
+    group_item_label: "Glean Validation Foreground Count"
+  }
+
+  measure: glean_validation_foreground_count {
```

Comment:
Some of these have a `count` suffix and some don't. We should ensure they all have the `count` suffix so it's obvious what is being plotted.

Reply:
This is the name of the actual metric, I don't think they should be suffixed with count (this would be glean_validation_foreground_count_count to be consistent with the naming convention).

Reply:
I do see the inconsistency between difference measures after looking into it a bit closer, I'll do it one way or the other.